pkg/auth/service_test.go

```diff
@@ -1259,11 +1262,13 @@ func TestAPIAuthService_WritePolicy(t *testing.T) {
 			err := s.WritePolicy(ctx, &model.Policy{
 				DisplayName: tt.policyName,
 				CreatedAt:   creationTime,
-				Statement: []model.Statement{{
-					Action:   tt.firstStatementAction,
-					Effect:   tt.firstStatementEffect,
-					Resource: tt.firstStatementResource,
-				}},
+				Statement: []model.Statement{
```

Comment:
Please avoid unnecessary whitespace changes.  They bloat a PR that is already too big, and make reviewing harder.  It is perfectly to fine to have a whitespace-only PR before or after a large PR, of course.